### DISCUSSION TITLE
plot curve tensorboard

### DISCUSSION THREAD

**Discussion by Bouhafs-Aek** [open-mmlab/mmaction2#2811] [General]
on tensorbaord plots , we have validation top1  and top5 , but training is per steps in separate graph. I need it by epoch
 training and validation

**Comment by LiYH1234** [comment]
Do you know how to draw this graph? I'm using mmaction2-1.0.0 version and always encountering issues. Do you know the correct command?

**Comment by LiYH1234** [comment]
> on tensorbaord plots , we have validation top1 and top5 , but training is per steps in separate graph. I need it by epoch training and validation

Do you know how to draw this graph? I'm using mmaction2-1.0.0 version and always encountering issues. Do you know the correct command?

**Comment by LiYH1234** [comment]
> on tensorbaord plots , we have validation top1 and top5 , but training is per steps in separate graph. I need it by epoch training and validation

这个top1和top5的图怎么画的呢，能分享我一下吗
